composer.json

```diff
@@ -22,11 +22,11 @@
     },
     "require": {
         "php": ">=5.6.0 || >=7.0",
-        "locomotivemtl/charcoal-app": "~0.8",
-        "locomotivemtl/charcoal-core": "~0.3",
-        "locomotivemtl/charcoal-factory": "~0.4",
-        "locomotivemtl/charcoal-object": "~0.4",
-        "locomotivemtl/charcoal-translator": "~0.3"
+        "charcoal/app": "~0.8",
+        "charcoal/core": "~0.3",
+        "charcoal/factory": "~0.4",
+        "charcoal/object": "~0.4",
+        "charcoal/translator": "~0.3"
```

Comment:
Update version numbers to correspond to the new versioning of [charcoal/charcoal](https://github.com/charcoalphp/charcoal).

Not sure how strict or loose it should be. For example:

* `"charcoal/app": ">=1.0.0"`
* `"charcoal/app": "^4.0.0"`
* `"charcoal/app": ">=1.0.0 <5.0.0"`

/cc @JoelAlphonso